File: Solutions/Easy/1437. Check If All 1's Are at Least Length K Places Away/Solution.cs
public class Solution {
    public bool KLengthApart(int[] nums, int k) {
        int pointer = -k - 1;
        for(int i = 0; i < nums.Length; i++){
            if(nums[i] == 1){
                if(pointer + k >= i) return false;
                pointer = i;
            }
        }
        return true;
    }
}

File: Solutions/Easy/1437. Check If All 1's Are at Least Length K Places Away/Solution.py
class Solution:
    def kLengthApart(self, nums: List[int], k: int) -> bool:
        pointer: int = -k - 1
        for i in range(len(nums)):
            if(nums[i] == 1):
                if(pointer + k >= i):
                    return False
                pointer = i
        return True
